autoload/yankitute.vim

```diff
@@ -75,3 +77,11 @@ function! s:eval(results, replace) abort
     call add(a:results, eval(a:replace[2:]))
     return submatch(0)
 endfunction
+
+function! s:expand_returns(lst) abort
```

Comment:
Mind renaming this to `s:expand_carriage_returns`? The current name is a bit ambiguous - first time I saw it I thought it was for splitting a value-to-be-returned...

Reply:
Also - `lst` and `arr` are **bad** names. All they tell us is that one is a list and the other is an array, which is really meaningless considering lists and arrays are the same thing in Vimscript.

Please name them `a:source` and `l:result` or something like that...